### ISSUE TITLE
Highight feature

### ISSUE THREAD

**Issue by MaeveS2** [Java-Discord/JavaBot#354]
# Highlight
## Description
A user has a list of highlighted words, and gets a DM by the bot when one of those words are sent in a message. The DM should contain the word that was mentioned, when it was sent and the channel is was sent in. Additionally it should have a button to jump to the message and contain a few messages sent before for context. 
## Syntax
* `/highlight add [word: String]` - adds a highlight
* `/highlight remove[word: String]` - removes a highlight
* `/highlight list` - shows a list of a users highlights
## Motivation
There are several reasons why this feature is useful, for example one could highlight their name, or a topic you're interested in.

**Comment by danthe1st** [comment]
This is already possible using TeXit's `,notify` command.

**Comment by DynxstyGIT** [comment]
yeah, but we obviously want our own implementation. Otherwise, we could’ve just used MEE6 all the time 😵‍💫

**Comment by MaeveS2** [comment]
It's not very useful if it's disabled is it
Also temporarily is synonymous with permanent
